src/components/fab/ExchangeFab.js

```diff
@@ -10,6 +9,7 @@ import FloatingActionButton from './FloatingActionButton';
 import { enableActionsOnReadOnlyWallet } from '@rainbow-me/config/debug';
 import Routes from '@rainbow-me/routes';
 import { magicMemo } from '@rainbow-me/utils';
+import watchingAlert from '@rainbow-me/utils/watchingAlert';
```

Comment:
Only comment is to expose watchingAlert in utils/index and then import it here as `import { watchingAlert } from '@rainbow-me/utils';`